pkger/clone_resource.go

```diff
@@ -278,3 +314,11 @@ func variableToResource(v influxdb.Variable, name string) Resource {
 
 	return r
 }
+
+func v1ColorsToViewColors(clrs []string) colors {
```

Comment:
thsi name is a bit misleading. you aren't returning ViewColors here, just colors which is different 🤔